app/views/home/index.html.erb

```diff
@@ -89,6 +89,19 @@
         <% end %>
       <% end %>
     <% end %>
+
+    <%= render Layout::GridComponent.new(
+          cols: 2,
+          classes: 'bg-blue-light rounded-12 p-4 mb-4',
+          cell_classes: 'my-auto'
+        ) do |grid| %>
+      <%= grid.with_feature_card do |feature| %>
+        <%= feature.with_header(title: t('home.audits.title'), level: 3) %>
+        <%= feature.with_description { t('home.audits.description_html', count: School.visible.count) } %>
+        <%= feature.with_button(t('home.enrol_your_school'), 'energy-audits', style: :primary) %>
```

Comment:
not sure this is the intended final link. Can you update to /find-out-more for now?